src/behaviours/update-app/installing-update-using-tray.test.ts

```diff
@@ -96,7 +104,9 @@ describe("installing update using tray", () => {
         expect(showApplicationWindowMock).not.toHaveBeenCalled();
       });
 
-      xit("notifies the user that checking for updates is happening", () => {});
+      it("notifies the user that checking for updates is happening", () => {
+        expect(showNotificationMock).toHaveBeenCalledWith("Checking for updates...");
```

Comment:
Given that we are about to remove this, is it worth it to test notifications?

Reply:
All the behaviours should be tested. If there is an implementation, there should be some unit test that cover it. 

If everything was done like that and we want to remove something, we would remove the implementation and check for failing unit tests and be absolutely sure that we didn't break anything.